src/connectedmachine/azext_connectedmachine/aaz/latest/connectedmachine/license/_update.py

```diff
@@ -13,19 +13,18 @@
 
 @register_command(
     "connectedmachine license update",
-    is_preview=True,
 )
 class Update(AAZCommand):
-    """Update operation to create or update a license.
+    """Update operation to update a license.
 
     :example: Sample command for license update
-        az connectedmachine license update --name 'myESULicense' --resource-group 'ytongtest' --license-type 'ESU' --state 'Deactivated' --target 'Windows Server 2012' --edition 'Datacenter' --type 'pCore' --processors 16 --subscription subscriptionId
+        az connectedmachine license create --name licenseName --resource-group myResourceGroup --location 'eastus2euap' --license-type 'ESU' --state 'Deactivated' --target 'Windows Server 2012' --edition 'Datacenter' --type 'pCore' --processors 16 --subscription mySubscription
```

Comment:
please check whether the command here is correct or not, as it is update operation.
```suggestion
        az connectedmachine license update --name licenseName --resource-group myResourceGroup --location 'eastus2euap' --license-type 'ESU' --state 'Deactivated' --target 'Windows Server 2012' --edition 'Datacenter' --type 'pCore' --processors 16 --subscription mySubscription
```